.github/scripts/create_and_push_docker_image.sh

```diff
@@ -15,50 +15,28 @@
 # limitations under the License.
 #
 
-INPUT_IMAGE_NAME=$1
-INPUT_IMAGE_TAG=$2
-INPUT_REPO_DIR=$3
-INPUT_NOTEBOOK_USER=$4
+INPUT_DOCKER_FULL_IMAGE_NAME=$1
+INPUT_PUSH=$2
 
-# Check for docker image name
-if [ -z "$INPUT_IMAGE_NAME" ]; then
-  echo "Input the image name for the docker image as the first parameter"
-  exit 1
-fi
+# ghcr.io/projectnessie/nessie-binder-demos:e12478cfb3447af4bc26e535a9daffe374233aa1
+DOCKER_IMAGE_NAME=$(echo "${INPUT_DOCKER_FULL_IMAGE_NAME}" | cut -f1 -d":")
+DOCKER_IMAGE_TAG=$(echo "${INPUT_DOCKER_FULL_IMAGE_NAME}" | cut -f2 -d":")
 
-# Check for docker tag
-if [ -z "$INPUT_IMAGE_TAG" ]; then
-  echo "Input the image tag for the docker image as the second parameter"
-  exit 1
+if [ "$INPUT_PUSH" = 'push' ]; then
```

Comment:
Do we need to check first if the second argument is set before the string comparison?

Reply:
do we? right now the param is passed in from all callers.
and even if it is not this test would just be false (and so no pushing is done by default, which is fine?)

Reply:
Yeah all good, I was just not sure about it 👍